src/streams/StreamItem.js

```diff
@@ -75,6 +90,21 @@ export default function StreamItem(props: Props): Node {
     onSwitch,
   } = props;
 
+  const showActionSheetWithOptions: ShowActionSheetWithOptions = useActionSheet()
+    .showActionSheetWithOptions;
+  const _ = useContext(TranslationContext);
+  const dispatch = useDispatch();
+  const backgroundData = useSelector(state => ({
+    auth: getAuth(state),
+    ownUser: getOwnUser(state),
+    streams: getStreamsById(state),
+    streamsByName: getStreamsByName(state), // TODO might need to pass stream by id here.
```

Comment:
Is this TODO still applicable?

Reply:
Also, it looks like `streamsByName` isn't needed for `backgroundData`, only for finding `stream` (and thereby the stream's ID.) So it'd be a bit cleaner to have that outside of `backgroundData`.

That would also then be helpful when we go on to have `StreamItem`'s parent pass it the stream ID, rather than making `StreamItem` look it up; if `streamsByName` is on its own outside `backgroundData`, it'll be easier then to see that it's no longer needed and cut it out.

Reply:
removed streamsByName.